database/migrations/Social/2023_02_04_234119_create_user_messages_table.php

```diff
@@ -14,7 +14,8 @@ class CreateUserMessagesTable extends Migration
     public function up()
     {
         Schema::connection('social')->create('user_messages', function (Blueprint $table) {
-            $table->char('messages_id', 36);
+            $table->increments('id');
```

Comment:
@FredPeal you dont need ID, we can use compound index

Reply:
We need the id field, because in the previous version for social , we have id. And use that for the relationship for user_messages_activites table